package.json

```diff
@@ -5,8 +5,10 @@
   "license": "MIT",
   "publisher": "serenade",
   "repository": "https://github.com/serenadeai/code",
-  "version": "1.5.3",
-  "extensionKind": "ui",
+  "version": "1.5.4",
+  "extensionKind": [
```

Comment:
I changed this to an array because of a warning from webpack while running `npm run build`